packages/state-transition/src/util/fulu.ts

```diff
@@ -13,7 +13,9 @@ export function initializeProposerLookahead(state: CachedBeaconStateElectra): Va
   const lookahead: ValidatorIndex[] = [];
 
   for (let i = 0; i < MIN_SEED_LOOKAHEAD + 1; i++) {
-    lookahead.push(...computeProposerIndices(ForkSeq.fulu, state, currentEpoch + i));
+    const epoch = currentEpoch + i;
+    const shuffling = state.epochCtx.getShufflingAtEpoch(epoch);
```

Comment:
this is called when we `upgradeStateToFulu()` and we don't have shuffling for the next epoch
needs to make sure we have it with similar pattern in `processProposerLookahead` above

Reply:
we only call this for current epoch and next epoch which seems fine here, are you suggesting to use the shuffling cache and call `getSync`?

Reply:
this looks good, I don't think we can do better in terms of performance
the only thing I want is to have same pattern for both places as commented [here](https://github.com/ChainSafe/lodestar/pull/7988#discussion_r2218119516)